util/tree/QueryByExample.ts

```diff
@@ -46,148 +62,144 @@ class Branch {
 // Internal state of query string generation
 class PathBuilderState {
 
-    private isMatch: boolean
-    private simplePredicates = ""
-    private complexPredicates = ""
-    private rootExpression: string
+    private isMatch: boolean;
+    private simplePredicates = "";
```

Comment:
I was just fixing tslint issues, in this case adding semicolons, so I'm not sure what the intent here is, if any.